### PR TITLE
docs: adjust the GitHub link on the header

### PR DESCRIPTION
**Pull request by yoshinorin**
## Purpose

The GitHub link on the header is broken if the display size is less than `1024` px. This PR is fix it.

### Before (less than `1024px`)

before

### After fix (less than `1024px`)

after

### Before (greater than `1024px`)

before-desk

### After fix (greater than `1024px` : Nothing changes)

after-desk

---

Thank you :)

### CODE REVIEW

File: website/siteConfig.js [hunk 1]
```diff
@@ -18,8 +18,7 @@ const siteConfig = {
   headerLinks: [
     {doc: 'getting-started', label: 'Docs'},
     {href: `/cats-effect/api/3.x/cats/effect/index.html`, label: 'API'},
-    {},
-    {href: 'https://github.com/typelevel/cats-effect', label: "GitHub", class: "github", external: true},
+    {href: 'https://github.com/typelevel/cats-effect', label: "GitHub", external: true},
```
Comment (yoshinorin) [inline website/siteConfig.js:21]: I read the docusaurus v1 document. The `class` field seems invalid. So, I delete it.

> https://v1.docusaurus.io/docs/en/site-config.html#headerlinks-array

File: website/static/css/custom.css [hunk 2]
```diff
@@ -13,27 +13,17 @@
...
-@media only screen and (min-width: 1024px) {
-}
-
-@media only screen and (max-width: 1023px) {
-}
-
-@media only screen and (min-width: 1400px) {
-}
-
-@media only screen and (min-width: 1500px) {
```
Comment (yoshinorin) [inline website/static/css/custom.css:28]: above `@media` rules are unnecessary.

File: website/static/css/custom.css [hunk 3]
```diff
@@ -13,27 +13,17 @@
...
+header nav li {
+  text-align: center;
 }
 
 header nav li:last-child a {
   width: 32px;
   height: 32px;
-  background-image: url(/cats-effect/img/github.png);
-  background-size: 32px 32px;
+  background: url("data:image/svg+xml;charset=utf-8,%3Csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='%23fff' d='M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12'/%3E%3C/svg%3E") no-repeat;
```
Comment (yoshinorin) [inline website/static/css/custom.css:23]: Convert from `.png` to `base64 svg`.
I referred to the https://docusaurus.io/ v2 header.

**Comment by djspiewak** [comment]
Oh, this conflicted with another open PR but it seems to do even more! Would you mind merging with the latest? Thank you so much for taking this on!

**Comment by yoshinorin** [comment]
@djspiewak 
Just now I resolved conflict & force pushed. Thank you :)

**Review by djspiewak — APPROVED** [review]

**Pull request merged**